cookbook/purge/purge_version_entities.rst

```diff
@@ -0,0 +1,107 @@
+How to customize the version purger to your needs
```

Comment:
"How to customize the version purger" or "How to adapt the version purger to your needs"